### PR TITLE
Add DTQP to WEIS

### PR DESCRIPTION
**Pull request by dzalkind**
This PR should follow #147 

## Purpose
Add DTQP module to WEIS, example for computing open-loop optimal control for an example (13_DTQP) using the weis_driver.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing
Example 13_DTQP will run a brief DTQP optimization.  Needs to be added to testing.

## Checklist
_Put an `x` in the boxes that apply._

- [ ] I have run existing tests which pass locally with my changes
- [ ] I have added new tests or examples that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: examples/13_DTQP/IEA-15-floating.yaml [hunk 1]
```diff
@@ -0,0 +1,1206 @@
+name: IEA 15MW Offshore Reference Turbine, with taped chord tip design
```
Comment (johnjasa) [inline examples/13_DTQP/IEA-15-floating.yaml:1]: Does this file need to be here as opposed to using the examples/06 version? Specifically, is this yaml different than that one?

Comment (johnjasa) [reply examples/13_DTQP/IEA-15-floating.yaml:1]: If it's the same, I'd suggest removing this one and referencing a singular geometry yaml.

**Review by johnjasa — APPROVED** [review]
Looks good! Glad to finally get this into WEIS, great efforts all!

File: examples/13_DTQP/IEA-15-floating.yaml (hunk 1 above)
Comment (dzalkind) [reply examples/13_DTQP/IEA-15-floating.yaml:1]: It should be the same as the geometry in example 12, so I agree, referencing a singular geometry yaml is best. Pushed!

**Comment by johnjasa** [comment]
Regarding the coveralls drop, is it reasonable to add the DTQP example to the test suite? Just adding that 13 example? Or is it unreasonably long and unable to be shortened?

**Pull request merged**